### PR TITLE
Make e2e-tests/scripts/update-chromedriver work on Mac.

### PR DESCRIPTION
**Pull request by dskloetd**
# Motivation

e2e-tests/scripts/update-chromedriver currently doesn't work on Mac.

# Changes

1. Call `/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version` if `google-chrome` doesn't exist.
2. Use `[[:space:]]` in sed instead of `\s`. Supposedly the latter only work on GNU sed.

# Tests

1. Used it manually on Mac.
2. Tested the part to determine chrome_major_version on my personal Linux laptop.

### CODE REVIEW

File: e2e-tests/scripts/update-chromedriver [hunk 1]
```diff
@@ -1,4 +1,13 @@
 #!/usr/bin/env bash
 set -euxo pipefail
+
+if command -v google-chrome; then
+  chrome_version=$(google-chrome --version)
+else
+  chrome_version=$(/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version)
```
Comment (bitdivine) [inline e2e-tests/scripts/update-chromedriver:7]: Try this:
```
osascript -e 'get version of application "Google Chrome"'
```
Might be better than assuming the default installation path.

Comment (bitdivine) [reply e2e-tests/scripts/update-chromedriver:7]: Update:  Tested on my intel mac.  It works there.  Not tested on M1/M2.

Comment (dskloetd) [reply e2e-tests/scripts/update-chromedriver:7]: Thanks. It works on my M1 mac as well.
Is this only for Mac or could this even replace the whole `if`?

Comment (dskloetd) [reply e2e-tests/scripts/update-chromedriver]: Done

Comment (bitdivine) [reply e2e-tests/scripts/update-chromedriver]: Only for mac.  osascript is applescript.  Played with it many years ago, it's quite a neat tool for running apple programs by remote control.  But it's also quite limited.

**Review by bitdivine — APPROVED** [review]

**Pull request merged**